Don't re-run the setup dialog each time the program starts
--- a/src/configwizard.cpp
+++ b/src/configwizard.cpp
@@ -753,7 +753,7 @@
 
             nextButtonClicked();
         #else
-            m_pageOffset += 7;
+            m_pageOffset += 8;
             BaseWizard::nextButtonClicked();
         #endif
 
